docs/guides/logdebug/1241-dagger-cloud-debugging.md

```diff
@@ -8,20 +8,17 @@ displayed_sidebar: '0.2'
 <p><a href="https://dagger.cloud/" target="_blank" rel="external"> Dagger Cloud </a> is under development, but we have just released the first telemetry feature!</p>
 
 :::tip
-Ensure you're using Dagger Engine/CLI version `0.2.18` or higher for Dagger Cloud.
-
-:::tip
-Ensure you're using Dagger Engine/CLI version `0.2.18` or higher for Dagger Cloud.
+Ensure you are using `dagger` CLI version `0.2.18` or higher for Dagger Cloud.
 :::
 
-To take advantage of this feature, you will need to create an account by following the steps below:
+To take advantage of this feature, you will need to create an account by following these steps:
 
 1. Initiate a login process using the CLI with `dagger login`.
 2. A new window will open in the browser requesting to sign-up using GitHub.
 3. After authenticating, authorize the Dagger GitHub application to finish the process.
-4. The browser window will close automatically and your CLI will be authenticated. Now,`dagger do` executions will now be listed in <a href="https://dagger.cloud/" target="_blank" rel="external">Dagger Cloud</a>.
+4. The browser window will close automatically and your CLI will be authenticated.
```

Comment:
Actually, this is not what's happening when you get authenticated for the first time. We display a landing page where users can either sign-in to the cloud with a CTA or close their tab and go back to the CLI

![Capture d’écran 2022-07-08 à 16 23 59](https://user-images.githubusercontent.com/1186424/178011504-b995a54f-c8eb-468b-ac18-9716844f3000.png)
.

Reply:
How about now?

Reply:
🚢